siriuspy/siriuspy/sofb/bpms.py

```diff
@@ -621,22 +743,57 @@ def calc_sp_multiturn_pos(self, **kwargs):
             v = v.reshape(-1, wsize)[:, mask]
             vs[a] = _np.std(v, axis=1)
 
-        s1, s2 = vs['A'] + vs['B'], vs['D'] + vs['C']
+        s1, s2 = vs['A'] + vs['C'], vs['D'] + vs['B']
```

Comment:
These short-named variables do not comply with pylint. We have to discuss. I think we should continue using pylint but in some cases, like variable names in specific mathematical expressions, long names do not help, short names sometimes are even preferably from a mathematical standpoint. Maybe we can adopt one of the solutions:

- Turn off this pylint rule (A pitty, since in other situations I think having longer variable names help code readability)
- In my code setup, prepending an underscore seems to circumvent this rule. We could use this judiciously, for math expressions where the meaning of a variable is explicit by the context.

take a look at
https://stackoverflow.com/questions/21833872/why-does-pylint-object-to-single-character-variable-names

Reply:
I'll change these variable names. I think it's easier now, but I see and agree with your point. Maybe we have a third option, which is to configure pylint to do not bother us for some specific variable names (2nd answer of the stackoverflow question you posted).

Reply:
done!

Reply:
on this issue, @murilobalves sent me yesterday a link to a ms code module that we could use to syncronize editor settings...

Reply:
interesting!